iis-wf/iis-wf-import/src/main/java/eu/dnetlib/iis/wf/importer/converter/AbstractEncodingAwareAvroConverter.java

```diff
@@ -0,0 +1,39 @@
+package eu.dnetlib.iis.wf.importer.converter;
+
+import eu.dnetlib.iis.wf.importer.input.approver.ResultApprover;
+
+/**
+ * Encoding aware avro converter.
+ * @author mhorst
+ *
+ * @param <T>
+ */
+public abstract class AbstractEncodingAwareAvroConverter<T> extends AbstractAvroConverter<T> {
```

Comment:
Well, you have created this abstract class just to hold encoding. The design is more complicated and you have another abstract class just to hold one variable (and the contract does not enforce using it in any way). I suggest that you get rid of this method.